experimental/dds/tree2/src/test/domains/schemaBuilder.spec.ts

```diff
@@ -248,10 +248,10 @@ describe("domains - SchemaBuilder", () => {
 			const z: number | undefined = x.recursive?.recursive?.number;
 		}
 
-		const inner = recursiveObject.create({ recursive: undefined, number: 5 });
+		const innerContents = { recursive: undefined, number: 5 };
+		const inner = recursiveObject.create(innerContents);
 		const testOptional = recursiveObject.create({ number: 5 });
-
-		const outer1 = recursiveObject.create({ recursive: inner, number: 1 });
+		const outer1 = recursiveObject.create({ recursive: innerContents, number: 1 });
```

Comment:
This test was re-using `inner`, causing it to be extracted twice. That's not allowed anymore. The inputs to factory functions are safe to re-use (`innerContents`) but the proxies that they now return should not be used to build tree content more than once.

Reply:
Is regular "multiple parents are disallowed" or a special?

Ex: would this be allowed? 
```ts
const foo = Foo.create({ bar: Bar.create(..) });

// detach the inner node
const bar = foo.bar;
foo.bar = undefined;

// reuse the inner node in new content
const baz = Foo.create({ bar });
```

Reply:
That's a really interesting question. I hadn't considered this. That would be cool to support. I think it would be possible by copying the data from `bar`, but still recognizing that it's a proxy and "rehydrating" it after the insert with the new edit node. Probably something we can try to do after initial "super factory." support